test.sh

```diff
@@ -120,6 +114,7 @@ if [[ "$RUN" =~ "start" ]] ; then
   done
   if [[ $I = 100 ]]; then
     echo "Boulder did not come up after ./start.py."
+    exit 1
```

Comment:
also: make start.up failing an actual error, since otherwise testing it here is pointless.

Reply:
This appears to have been broken since we moved to python3.